presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the `http-server.https.excluded-cipher` config
+  property to match Jetty's default excluded cipher suites. Specifically, all
+  ciphers without forward secrecy, and all ciphers that use the RSA key exchange
+  algorithm are excluded by default. Consequently, TLS 1.0 or TLS 1.1 are no
+  longer supported by default. `http-server.https.excluded-cipher` config property
+  can be set to empty string to restore the old behavior.
+* Allow running on a JVM from any vendor that meets the functional requirements.
+* Add :func:`ST_GeomFromBinary` and :func:`ST_AsBinary` functions that converts
+  geometries to and from Well-Known Binary format.
+* Remove the ``verbose_stats`` session property. Rename the ``task.verbose-stats``
+  configuration property to ``task.per-operator-cpu-timer-enabled``.
+* Remove the ``experimental.enable-new-stats-calculator`` configuration property.
+* Improve query planning performance for queries containing multiple joins
+  and a large number of columns (:issue:`11196`).
+
+Hive Connector Changes
+----------------------
+
+* Fix coordinator OOM when a query scans many partitions of a Hive table (:issue:`11322`).
+* Improve readability of columns, partitioning, and transactions in explain plains.
+
+Resource Group Changes
+----------------------
+
+* Add documentation for new resource group scheduling policies.
+* Add built-in file based session property manager to automate the setting
```

Comment:
Let's move the link to this item (you'd expect to find it here since that's where it is introduced)